crash-reporting-core/src/commonMain/kotlin/dev/icerock/moko/crashreporting/core/CrashReportingCore.kt

```diff
@@ -0,0 +1,22 @@
+/*
+ * Copyright 2020 IceRock MAG Inc. Use of this source code is governed by the Apache 2.0 license.
+ */
+
+package dev.icerock.moko.crashreporting.core
+
+class CrashReportingCore {
```

Comment:
it should be just object, it not store any data and instantiation for each time will be bad